src/components/dotMenu/DotMenu.js

```diff
@@ -0,0 +1,70 @@
+/**
+ *  @author aramsey
+ *
+ *  A component which displays 3 vertical dots, which when
+ *  clicked will display a menu.
+ *
+ *  MenuItems should be passed into the render prop which in turn
+ *  will provide a function for closing the menu.
+ *
+ *  Sample usage:
+ *  <DotMenu
+ *    baseId="myId"
+ *    render={(onClose) => [
+ *        <MenuItem
+ *          onClick={onClose}>
+ *          Click Me
+ *        </MenuItem>,
+ *        ...
+ *        ]}
+ *  />
+ **/
+
+import React, { useState } from "react";
+
+import { Menu, IconButton } from "@material-ui/core";
+import MoreVertIcon from "@material-ui/icons/MoreVert";
+
+import buildID from "components/utils/DebugIDUtil";
+
+function DotMenu(props) {
```

Comment:
This also does not look like the latest version of this file, which was updated in cyverse-de/ui-lib#52